scripts/utils/data_model/xcm_json_model.py

```diff
@@ -0,0 +1,145 @@
+import math
+from typing import List, Optional
+
+from .base_model import BaseObject
+from .chain_json_model import Chain
+from PyInquirer import prompt
+from utils.fee_calculation_functions import biforst_base_fee, heiko_base_fee, kintsugi_base_fee, karura_base_fee, turing_base_fee, moonriver_fee_calculation
+
+from utils.questions import coefficient_question
+
+
+class FeeMode(BaseObject):
+    type: str
+    value: str
+
+    def __init__(self, type, value: Optional[str] = None):
+        self.type = type
+        if value is not None:
+            self.value = value
+
+
+class Fee(BaseObject):
+    mode: FeeMode
+    instructions: str
+    asset: dict
+
+    def __init__(self, mode: FeeMode, instructions, asset=None):
+
+        if (isinstance(mode, FeeMode)):
+            self.mode = mode
+        else:
+            self.mode = FeeMode(**mode)
+
+        self.instructions = instructions
+
+        if asset is not None:
+            self.asset = asset
+
+    def calculate_fee(self, network: Chain, xcm_asset):
+
+        def base_fee_and_multiplier(base_fee):
+            coefficient_input = prompt(coefficient_question())
+            coefficient = coefficient_input.get('coefficient')
+
+            return math.ceil(base_fee * coefficient)
```

Comment:
This may lead to slight overestimate because in our case we represent coefficient as decimal (but rational representation is used in substrate) and rounding up (floor is used in substrate). But I think we can leave as it is due to the fact that we don't have rational numbers implementation